### PR TITLE
Adding error condition to billing check

### PR DESCRIPTION
**Pull request by markmirch**
## Description

Additional error check on billing enabled function

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Enviroment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Enviroment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: functions/billing/main.py [hunk 1]
```diff
@@ -104,6 +104,9 @@ def __is_billing_enabled(project_name, projects):
     try:
         res = projects.getBillingInfo(name=project_name).execute()
         return res['billingEnabled']
```
Comment (ace-n) [inline functions/billing/main.py:106]: Would it be cleaner to use `return 'billingEnabled' in res and res['billingEnabled']`?

Comment (markmirch) [reply functions/billing/main.py:106]: Yes, though I feel like that's slightly harder to understand.

Comment (busunkim96) [reply functions/billing/main.py:106]: I would be alright with keeping it as is. This style is fairly common in Python. "Easier to ask for forgiveness than permission".

https://docs.python.org/3/glossary.html?highlight=eafp

> Easier to ask for forgiveness than permission. This common Python coding style assumes the existence of valid keys or attributes and catches exceptions if the assumption proves false. This clean and fast style is characterized by the presence of many try and except statements. The technique contrasts with the LBYL style common to many other languages such as C.

Comment (ace-n) [reply functions/billing/main.py:106]: Ah, thanks for clarifying @busunkim96! (I guess Node has taught me to optimize for brevity. 😄 )

**Pull request merged**